lib/razorpay/utility.rb

```diff
@@ -0,0 +1,23 @@
+require 'razorpay'
+require 'openssl'
+
+# Helper functions are defined here
+module Razorpay
+  class Utility
+    def self.validate_payment_signature(order_id, payment_id, signature)
```

Comment:
We're changing this back to accept an `attributes` hash as the only param, to avoid exactly this case.